source/jobs/recover-jobs.txt

```diff
@@ -0,0 +1,49 @@
+.. _rm-sync-jobs-recoverability:
+
+==================
+Recover a Sync Job
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can recover continuous (CDC) sync jobs that 
+have failed within a 24-hour period. Procedures 
+for recovering a CDC sync job depend on if your using the embedded 
+or Kafka deployment mode.
+
+Embedded Mode Recovery
+----------------------
+
+For embedded mode recovery, the process to recover
+a sync job is manually triggered. Navigate to the 
+:guilabel:`Data Migration` tab and click 
+:guilabel:`Resume` on the failed CDC sync job. 
+
```

Comment:
(information typing)
I'd like to see this look a little more like a traditional task page. I could see moving these two descriptions into tabs, perhaps. Happy to chat if you like!

Reply:
Updated.

Reply:
Thanks a ton @ianf-mongodb! I appreciate the thoughtfulness with which you approached this!